pydy/codegen/ode_function_generators.py

```diff
@@ -641,16 +641,26 @@ class CythonODEFunctionGenerator(ODEFunctionGenerator):
 
     def __init__(self, *args, **kwargs):
 
+        self._options = {'tmp_dir': None,
+                         'prefix': 'pydy_codegen',
+                         'cse': True,
+                         'verbose': False}
```

Comment:
I'll be able to give it more of a look over later today or possibly tomorrow. First look though it appears that you are adding more things that a user can set. If this is the case should this be added to a docstring or documentation somewhere? If I remember right the docstring is built dynamically for this class and may be difficult to edit.